src/components/molecules/PanelOptions/PanelOptions.tsx

```diff
@@ -14,6 +14,7 @@ export const PanelOptions = (props: PanelOptionsProps) => (
     <div className={'PanelOptions__box-buttons'}>
       <Link to={'/token'}>
         <BoxButton
+          className={'Panel CreateToken'}
```

Comment:
the name of the classes don't follow the `BEM` convention

Reply:
And what name should be used be in this case?

Reply:
How come 'Panel' doesn't follow the BEM convention?

Reply:
In my mind I think the block is `PanelOptions__box-buttons` and you have added a new element, so

B = block = PanelOptions__box-buttons
E =  element = `new_element`
M = modifier

What do you say? it's only a detail